src/main/java/appeng/fluids/client/gui/GuiFluidTerminal.java

```diff
@@ -229,22 +228,23 @@ protected void handleMouseClick( Slot slot, int slotIdx, int mouseButton, ClickT
 	{
 		if( slot instanceof SlotFluidME )
 		{
-			SlotFluidME meSlot = (SlotFluidME) slot;
+			final SlotFluidME meSlot = (SlotFluidME) slot;
+
 			if( clickType == ClickType.PICKUP )
 			{
 				// TODO: Allow more options

```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)